react-state/my-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import ToggleButton from './ToggleButton';
+import { ToggleLamberto } from './ToggleButton';
```

Comment:
Usually you would combine these two imports:
`import ToggleButton, { ToggleLamberto } from './ToggleButton';`